setenv

```diff
@@ -9,7 +9,7 @@ CITADEL_ROOT="$(dirname $(readlink -f "${BASH_SOURCE[0]}"))"
 alias citadel-update="${CITADEL_ROOT}/scripts/update/update"
 alias lncli="docker exec -it lnd lncli"
 alias bitcoin-cli="docker exec -it bitcoin bitcoin-cli"
-alias docker-compose="sudo docker compose"
-alias docker="sudo docker"
+alias docker-compose="docker compose -f docker-compose.citadel.yml"
```

Comment:
This seems to me like it's defeating the purpose of this change. If we add the path to the config in the alias then users will be able to run `docker-compose up` just like before, again running into the same issue.

Reply:
yeah, good point. That's just pointless IMO and should be remove. Let me fix that, thanks for catching it.

Reply:
Not sure if we can remove the `sudo docker` alias though, @AaronDewes can you confirm sudo is not needed for docker?

Reply:
tbh running `docker` as root is just a bad idea... (among the other reasons https://medium.com/@suixo/linux-users-running-docker-without-sudo-is-dangerous-3e5c5654abea) but in general users should add themselves to the `docker` group instead and just run commands that way.

Reply:
Erm... the article you linked states the opposite of what you are saying. And afaik the `citadel` user is added to the `docker` group which essentially makes it root anyway. There was a reason it was done this way though.

Reply:
no that's exactly what I'm saying, that running docker as root in general is a bad idea, so USUALLY the solution is adding only SOME users to the docker group, which is better than allowing anybody to run it via sudo.

But that's just my take.. as in "that's the least problematic solution, though still not a proper solution"

Hopefully that clarifies my POV but feel free to propose alternatives or argument why/how sudo is better than using the group approach

Reply:
I might have bad reading comprehension then.

You:
> running docker as root in general is a bad idea

Article title:
> running docker without sudo is dangerous

I'm not a security expert so I won't try to argue for either, I'm just saying there was a reason it was done this way afaik.

Reply:
no sorry, I wrote it in a rush and my sentence was confusing.
I linked the article as a POV but then stated my own which is kinda confirming the same danger but with a different solution, so you're reading is correct, so my bad 🤦 

MY POV: In general, once you can run Docker, you essentially CAN have access to root capabilities (unless under very specific circumstances/setups).

Here we are enforcing the use of sudo for no real added benefit, especially cause we don't know the user OS setup.

So my point here was, unless we have a compelling reason or even a NEED to have docker run with sudo, we should just let the user choose its own setup... cause Docker permission scheme is screwed anyway.

Reply:
Ok, gotcha.

> Here we are enforcing the use of sudo for no real added benefit, especially cause we don't know the user OS setup. So my point here was, unless we have a compelling reason or even a NEED to have docker run with sudo, we should just let the user choose its own setup... cause Docker permission scheme is screwed anyway.

When this code was written there was only Citadel OS (or Umbrel OS for that matter), so the setup was known and I vaguely recall this topic coming up before, I don't remember the explanation for running docker as sudo though, hence why I pinged Aaron. It might well be outdated since we have many users running on their own setup now and the code also changed a lot.